Lib/_pydatetime.py

```diff
@@ -1241,6 +1241,10 @@ def weekday(self):
         "Return day of the week, where Monday == 0 ... Sunday == 6."
         return (self.toordinal() + 6) % 7
 
+    def timestamp(self):
```

Comment:
Can we put it somewhere else instead of in the middle of the ISO methods? like after `toordinal()` as you did in the docs.